environments/prod/backend_lb_config.yaml

```diff
@@ -29,3 +30,25 @@ gateways:
       - product: pip
         component: account-management
         ssl_enabled: true
+    # Video Hearings
+      - product: vh
```

Comment:
do you need to add to prod now? is the application there and running?

you're welcome to add it but just want to discourage noise from failing health probes if the application is not there

Reply:
They are looking to do production cut over March so we are setting all environments up.